###############################################################################
# Copyright 2008 Sun Microsystems, Inc.  All rights reserved.
# Use subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the
# "Software"), to deal in the Software without restriction, including
# without limitation the rights to use, copy, modify, merge, publish,
# distribute, and/or sell copies of the Software, and to permit persons
# to whom the Software is furnished to do so, provided that the above
# copyright notice(s) and this permission notice appear in all copies of
# the Software and that both the above copyright notice(s) and this
# permission notice appear in supporting documentation.
# 
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
# OF THIRD PARTY RIGHTS. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
# HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL
# INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING
# FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
# WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
# 
# Except as contained in this notice, the name of a copyright holder
# shall not be used in advertising or otherwise to promote the sale, use
# or other dealings in this Software without prior written authorization
# of the copyright holder.
#

Search /usr/include/X11/bitmaps, then /usr/X11/include/X11/bitmaps, for
bitmaps in XmuLocatePixmapFile(), as specified in PSARC/2005/192.

diff -urp -x '*~' -x '*.orig' src/LocBitmap.c src/LocBitmap.c
--- src/LocBitmap.c	2006-11-08 09:51:59.000000000 -0800
+++ src/LocBitmap.c	2008-01-11 20:50:30.995077000 -0800
@@ -99,10 +99,15 @@ XmuLocatePixmapFile(Screen *screen, _Xco
 			    int *widthp, int *heightp, int *xhotp, int *yhotp)
 {
 
+#undef BITMAPDIR
 #ifndef BITMAPDIR
 #define BITMAPDIR "/usr/include/X11/bitmaps"
 #endif
 
+#ifndef BITMAPX11DIR
+#define BITMAPX11DIR "/usr/X11/include/X11/bitmaps"
+#endif
+
     Display *dpy = DisplayOfScreen (screen);
     Window root = RootWindowOfScreen (screen);
     Bool try_plain_name = True;
@@ -149,7 +154,8 @@ XmuLocatePixmapFile(Screen *screen, _Xco
      *    1.  name if it begins with / or ./
      *    2.  "each prefix in file_paths"/name
      *    3.  BITMAPDIR/name
-     *    4.  name if didn't begin with / or .
+     *    4.  BITMAPX11DIR/name
+     *    5.  name if didn't begin with / or .
      */
 
     for (i = 1; i <= 4; i++) {
@@ -182,6 +188,9 @@ XmuLocatePixmapFile(Screen *screen, _Xco
 	    XmuSnprintf(filename, sizeof(filename), "%s/%s", BITMAPDIR, name);
 	    break;
 	  case 4:
+	    XmuSnprintf(filename, sizeof(filename), "%s/%s", BITMAPX11DIR, name);
+	    break;
+	  case 5:
 	    if (!try_plain_name) continue;
 	    fn = (char *) name;
 	    break;
